source/client-side-encryption/tests/README.md

```diff
@@ -0,0 +1,3191 @@
+# Client Side Encryption Tests
+
+______________________________________________________________________
+
+## Introduction
+
+This document describes the format of the driver spec tests included in the JSON and YAML files included in the `legacy`
+sub-directory. Tests in the `unified` directory are written using the
+[Unified Test Format](../../unified-test-format/unified-test-format.md).
+
+The `timeoutMS.yml`/`timeoutMS.json` files in this directory contain tests for the `timeoutMS` option and its
+application to the client-side encryption feature. Drivers MUST only run these tests after implementing the
+[Client Side Operations Timeout](../../client-side-operations-timeout/client-side-operations-timeout.md) specification.
+
+Additional prose tests, that are not represented in the spec tests, are described and MUST be implemented by all
+drivers.
+
+Running spec and prose tests require that the driver and server both support Client-Side Field Level Encryption. CSFLE
+is supported when all of the following are true:
+
+- Server version is 4.2.0 or higher. Legacy spec test runners can rely on `runOn.minServerVersion` for this check.
+- Driver has libmongocrypt enabled
+- At least one of [crypt_shared](../client-side-encryption.rst#crypt_shared) and/or
+  [mongocryptd](../client-side-encryption.rst#mongocryptd) is available.
+
+## Spec Test Format
+
+The spec tests format is an extension of the
+[transactions spec legacy test format](../../transactions/tests/legacy-test-format.md) with some additions:
+
+- A `json_schema` to set on the collection used for operations.
+- An `encrypted_fields` to set on the collection used for operations.
+- A `key_vault_data` of data that should be inserted in the key vault collection before each test.
+- Introduction `autoEncryptOpts` to `clientOptions`
+- Addition of `\$db` to command in `command_started_event`
+- Addition of `\$\$type` to command_started_event and outcome.
```

Comment:
These dollar signs should not be escaped. Also applies to some content later in the file.